.github/workflows/stale.yml

```diff
@@ -23,14 +23,17 @@ jobs:
         days-before-stale: 30
         days-before-close: 5
         stale-issue-message: 'This issue is stale because it has been open 30 days with no activity. Please comment or update this issue or it will be closed in 5 days.'
-        stale-issue-label: $stale_label
+        stale-issue-label: '${{ env.stale_label }}'
         exempt-issue-labels: 'pinned, Fixed in next release, bug, never-stale, documentation, investigating'
         exempt-all-issue-assignees: true
         operations-per-run: 300
         close-issue-reason: 'not_planned'
 
-  remove_stale: # trigger "stale" removal immediately when stale issues are commented on
-    if: github.event_name == 'issue_comment'
+  remove_stale:
+    # trigger "stale" removal immediately when stale issues are commented on
+    # we need to explicitly check that the trigger does not run on comment on a PR as
+    # 'issue_comment' triggers on issues AND PR comments
+    if: github.event_name == 'issue_comment' && ${{ !github.event.issue.pull_request }}
```

Comment:
@yubiuser , oddly this seems to be running on the `schedule` trigger, too. (see https://github.com/pi-hole/docker-pi-hole/actions/runs/5505595159)

I feel the `if:` statement here should stop that from happening, but.... 🤔

Reply:
Yes, you are right. I'm not sure why it's running... the `if` clearly states that you need `issue_comment` to trigger this. Strange.